addons/test_mail_full/tests/test_sms_management.py

```diff
@@ -109,6 +109,15 @@ def test_sms_set_outgoing(self):
             {'partner': self.partner_2, 'number': self.notif_p2.sms_number, 'state': 'ready'}
         ], 'TEST BODY', self.msg, check_sms=False)    # do not check new sms as they already exist
 
+    def test_sms_notify_cancel(self):
```

Comment:
Note that as tests are executed alphabetically, we try to keep test order alphabetically (that way when having issues in a file, we can follow test order in the file).

Reply:
Yes, it might be useful, thanks